src/main/java/org/jenkinsci/plugins/buildnamesetter/BuildNameSetter.java

```diff
@@ -5,11 +5,13 @@
 import hudson.matrix.MatrixAggregatable;
 import hudson.matrix.MatrixAggregator;
 import hudson.matrix.MatrixBuild;
+import hudson.matrix.MatrixProject;
```

Comment:
While the plugin is currently on 1.480.x, what happens to this dependency when that baseline gets raised? Since Matrix Plugin has been detached from core a while back, is there a way for this PR to be implemented that allows the Matrix Plugin dependency to be made optional?